agents/langchain_search/main.py

```diff
@@ -0,0 +1,35 @@
+import os
+
+import fixieai
+import yaml
+from langchain.agents import initialize_agent
+from langchain.agents import load_tools
+from langchain.chat_models import ChatOpenAI
+from langchain.llms import OpenAI
+
+# Set up the agent in standalone mode.
```

Comment:
What does "standalone mode" mean? Is that a documented thing?

Reply:
Might be good to explain that this simply means that the agent does not provide any few shot examples and instead registers its main() func to be invoked whenever there is a query to the agent. The point is, this is not a recipe that we would want people to use for all of their agents, normally agents should be using CodeShots.